uniqueids/admin.py

```diff
@@ -1,6 +1,39 @@
 from django.contrib import admin
 
-# Register your models here.
 from .models import Record
 
-admin.site.register(Record)
+
+class RecordAdmin(admin.ModelAdmin):
+    list_display = [
+        "id", "identity", "write_to", "created_at", "updated_at"]
+    list_filter = ["write_to", "created_at"]
+    search_fields = ["identity", "write_to"]
+    actions = ["regenerate_personnel_code"]
+
+    def regenerate_personnel_code(self, request, queryset):
+        created = 0
+        skipped = 0
+        for record in queryset:
+            if record.write_to != "personnel_code":
+                skipped += 1
+                continue
+            Record.objects.create(
+                identity=record.identity, write_to="personnel_code",
+                length=record.length, created_by=request.user)
```

Comment:
I'm not sure if we want to create new codes? I think it might be better for this to be idempotent, so all that it does is resend existing codes.

I'm thinking of the case of where say we've been creating health IDs for a month, but only in the past week have we been sending the codes. If we select all from the past month to resend, then it will disable the codes that were sent in the last week. I think it would cause less confusion in those cases to just resend the SMS with the code, rather than sending them a new code.